projects/plugins/jetpack/_inc/client/components/section-header/index.tsx

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames';
+import Card from 'components/card';
+import React from 'react';
+
+import './style.scss';
+
+interface SectionHeaderProps {
+	label?: string;
```

Comment:
`NB`: Should we allow this to be optional? If so, maybe it should have a default value of an empty string?

Reply:
Thank you! [ba7f535](https://github.com/Automattic/jetpack/pull/26644/commits/ba7f535f684731b4bf5b491b4274301907aa37d9) looks good to me